.editorconfig

```diff
@@ -6,3 +6,6 @@
 
 root = True
 
+[*.kt]
```

Comment:
Please remove this. 🙂

Reply:
I was trying to avoid future indentation issues when the IDE config is different than the project one. Just out of curiosity, why not to use the editor config to unify it?

Reply:
Feel free to put up an issue for discussing about this. I don't think it's relevant to the PR you have right now, so it's best to keep it simple.

We also use ktlint, so you can have your editor follow that configuration [here](https://github.com/mozilla-mobile/android-components/#style) and [here](https://github.com/pinterest/ktlint#option-1-recommended).